spotty/providers/gcp/deployment/dm_templates/instance_template.py

```diff
@@ -53,6 +53,7 @@ def prepare_instance_template(instance_config: InstanceConfig, container: Contai
         'ZONE': instance_config.zone,
         'MACHINE_TYPE': instance_config.machine_type,
         'SOURCE_IMAGE': image_link,
+        'BOOT_DISK_SIZE': str(instance_config.boot_disk_size) if instance_config.boot_disk_size else '10',
```

Comment:
I think it's not gonna work if the user doesn't specify a boot disk size and the size for the source image is more than 10GB. In that case, we need to use the size of the original image, so the `diskSizeGb` parameter in the template should not be set at all. I'll fix it in the right way and release a new version, hopefully, today.